examples/docker-build/dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:10-stretch-slim as builder
+
+WORKDIR /src
+
+# Remove unnecessary files from node_modules (.md, .ts, ...)
+RUN cd /tmp && \
+  curl -#L https://github.com/tj/node-prune/releases/download/v1.0.1/node-prune_1.0.1_linux_amd64.tar.gz | tar -xvzf- && \
+  mv -v node-prune /usr/local/bin && rm -rvf * && \
+  echo "node-prune" > /usr/local/bin/node-clean && chmod +x /usr/local/bin/node-clean
+
+COPY package*.json ./
+RUN npm install --no-progress && node-clean
```

Comment:
Internally all of the nuxt projects are using yarn and it is included by default in official docker images. I suggest to use yarn as example + appreciate flags to prevent lock file modification.

Reply:
```
yarn install \
 --prefer-offline \
 --frozen-lockfile \
 --non-interactive \
 --production=false
```